inc/zoo/map/RobinHood.h

```diff
@@ -386,26 +337,34 @@ struct RH_Frontend_WithSkarupkeTail {
         auto swarIndex = index / MD::Lanes;
         auto intraIndex = index % MD::Lanes;
         auto mdp = this->md_.data() + swarIndex;
-        constexpr auto MaxRelocations = 64;
+        constexpr auto MaxRelocations = 100;
```

Comment:
If we relocate too may times...we throw an exception? Fall back to something else?

Reply:
We have to think deeper about whether or how to support the strong exception guarantee.
It is expensive to determine if the insertion will succeed, it is expensive to have a journal of operations to do to commit an insertion.
This is good to expose to the audience of the presentations.
Now, we have a solution, that is a compromise between the two options.

Reply:
I thought we had inplace insert implemented (though not tested well) and now journaled insert implemeted and tested.  Seeems like we should keep both functions around, and (if time allows) fix up insert and benchmark and present results for both.

Reply:
Sure, I like the 'optimistic' insertion implementation.